src/sentry/sentry_apps/metrics.py

```diff
@@ -74,6 +92,7 @@ class SentryAppEventType(StrEnum):
     EXTERNAL_ISSUE_CREATED = "external_issue.created"
     EXTERNAL_ISSUE_LINKED = "external_issue.linked"
     SELECT_OPTIONS_REQUESTED = "select_options.requested"
+    ALERT_RULE_ACTION_REQUESTED = "alert_rule_action.requested"
```

Comment:
Adding other external request event types since we're now converting the given event in [send_and_save_sentry_app_request](https://github.com/getsentry/sentry/pull/87212/files?diff=unified&w=1#diff-e8611831b9c153669ffdde7519f101fcf106d72201eb045606bf74370b9ee703R73) to this enum

Reply:
is this used in this PR?

Reply:
please click on the link in my comment, we're now converting/casting the event string so we need to make sure all expected events are present here.

Reply:
ok